src/Plaster.psd1

```diff
@@ -19,98 +28,33 @@
-GUID = 'cfce3c5e-402f-412a-a83a-7b7ee9832ff4'
-
-# Author of this module
-Author = 'Microsoft Corporation'
-
-# Company or vendor of this module
-CompanyName = 'Microsoft Corporation'
-
-# Copyright statement for this module
-Copyright = '(c) Microsoft Corporation 2016. All rights reserved.'
-
-# Description of the functionality provided by this module
-Description = 'Plaster scaffolds PowerShell projects and files.'
-
-# Minimum version of the Windows PowerShell engine required by this module
-PowerShellVersion = '5.0'
-
-# Name of the Windows PowerShell host required by this module
-# PowerShellHostName = ''
-
-# Minimum version of the Windows PowerShell host required by this module
-# PowerShellHostVersion = ''
-
-# Minimum version of Microsoft .NET Framework required by this module
-# DotNetFrameworkVersion = ''
-
-# Minimum version of the common language runtime (CLR) required by this module
-# CLRVersion = ''
-
-# Processor architecture (None, X86, Amd64) required by this module
-# ProcessorArchitecture = ''
-
-# Modules that must be imported into the global environment prior to importing this module
-# RequiredModules = @()
-
-# Assemblies that must be loaded prior to importing this module
-# RequiredAssemblies = @()
-
-# Script files (.ps1) that are run in the caller's environment prior to importing this module.
-# ScriptsToProcess = @()
-
-# Type files (.ps1xml) to be loaded when importing this module
-# TypesToProcess = @()
-
-# Format files (.ps1xml) to be loaded when importing this module
-# FormatsToProcess = @()
-
-# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
-# NestedModules = @()
-
-# Functions to export from this module - explicitly list each function that should be
-# exported.  This improves performance of PowerShell when discovering the commands in
-# module.
-FunctionsToExport = @(
-    'Invoke-Plaster'
-    'New-PlasterManifest'
-    'Test-PlasterManifest'
-    )
-
-# Cmdlets to export from this module
-# CmdletsToExport = '*'
-
-# Variables to export from this module
-# VariablesToExport = '*'
+        }
+    }
 
-# Aliases to export from this module
-# AliasesToExport = '*'
+    # Author of this module
+    Author = 'Microsoft Corporation'
 
-# DSC resources to export from this module
-# DscResourcesToExport = @()
+    # Company or vendor of this module
+    CompanyName = 'Microsoft Corporation'
 
-# List of all modules packaged with this module
-# ModuleList = @()
+    # Copyright statement for this module
+    Copyright = '(c) Microsoft Corporation 2016. All rights reserved.'
 
-# List of all files packaged with this module
-# FileList = @()
+    # Description of the functionality provided by this module
+    Description = 'Plaster scaffolds PowerShell projects and files.'
 
-# HelpInfo URI of this module
-# HelpInfoURI = ''
+    # Minimum version of the Windows PowerShell engine required by this module
+    PowerShellVersion = '5.0'
```

Comment:
I think we've got Appveyor currently running the test suite as v5. I'll have a look into it, see if i can figure it out but from memory I think it might be possible to do multiple passes to test on the different versions we want to target as supported.